utils.js

```diff
@@ -42,5 +42,7 @@ function messageResponse (name, message) {
 
 module.exports = {
   parseMessage,
-  messageResponse
+  messageResponse,
+  printGeneralHelp,
```

Comment:
Do we need to make this functions public? Can't we do this with `messageResponse` function?